### PR TITLE
add CONTRIBUTING.md and update README to reference it

### PR DESCRIPTION
**Pull request by elrayle**
### Description

Adds a file that describes the process for contributing to this repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE.md).
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## Prerequisites for running and testing code
+
+These are one time installations required to be able to test your changes locally as part of the pull request (PR) submission process.
+
+1. install Go [through download](https://go.dev/doc/install) | [through Homebrew](https://formulae.brew.sh/formula/go)
+1. [install golangci-lint](https://golangci-lint.run/usage/install/#local-installation)
```
Comment (jonabc) [inline CONTRIBUTING.md:19]: same here - this project doesn't use go

Comment (elrayle) [reply CONTRIBUTING.md:19]: Thanks.  We just caught these in the CONTRIBUTING_js.md template as well.  I'm working on how best to update the template, and the CONTRIBUTING.md files in the repos.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE.md).
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## Submitting a pull request
+
+1. [Fork][fork] and clone the repository
+1. Configure and install the dependencies: `npm install`
+1. Make sure the tests pass on your machine: `npm run test`
+1. Make sure linter passes on your machine: `npm run lint`
```
Comment (jonabc) [inline CONTRIBUTING.md:19]: nit: I've got this repo set up that `npm run test` will also run the linter, so this step shouldn't be necessary if the dev is also instructed to run the test command
```suggestion
```

**Review by jonabc — APPROVED** [review]
one nit but this would be fine to merge as is if you'd prefer to leave the lint instruction

File: CONTRIBUTING.md (hunk 2 above)
Comment (elrayle) [reply CONTRIBUTING.md]: Glad you caught that.

**Pull request merged**